charts/drone/templates/service.yaml

```
@@ -2,6 +2,8 @@ apiVersion: v1
 kind: Service
 metadata:
   name: {{ include "drone.fullname" . }}
+  annotations:
+    {{ toYaml .Values.service.annotations | nindent 4 }}
```

Without chomping the whitespace, you'll have an extra newline and 4 spaces.
```suggestion
    {{- toYaml .Values.service.annotations | nindent 4 }}
```